apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -411,6 +411,11 @@ function initializeBlocklyWrapper(blocklyInstance) {
   blocklyWrapper.Block.prototype.setStrictOutput = function (isOutput, check) {
     return this.setOutput(isOutput, check);
   };
+
+  blocklyWrapper.Block.prototype.setTitleValue = function (newValue, name) {
+    return this.setFieldValue(newValue, name);
+  };
```

Comment:
Thanks for adding this!

Reply:
Could you add a comment for why this is needed? I was also a little thrown off by the fact that we were calling 'setTitleValue' inside of a callback, but it makes sense with the context that field used to be called title 🙂

Reply:
just checking - should this read `"A block's 'field' used to be called its 'title'"` (rather than `"a field's title"`)?